src/main/java/com/esri/samples/displayinformation/sketch_on_map/SketchOnMapSample.java

```diff
@@ -0,0 +1,415 @@
+import com.esri.arcgisruntime.mapping.view.IdentifyGraphicsOverlayResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.mapping.view.SketchCreationMode;
+import com.esri.arcgisruntime.mapping.view.SketchEditor;
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Orientation;
+import javafx.geometry.Point2D;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.ComboBox;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+public class SketchOnMapSample extends Application {
+
+  private MapView mapView;
+  private SketchEditor sketchEditor;
+  private Button redoButton;
+  private Button undoButton;
+  private Button clearButton;
+  private Button saveButton;
+  private Button editButton;
+  private Button stopSketchButton;
+  private GraphicsOverlay graphicsOverlay;
+  private SimpleFillSymbol fillSymbol;
+  private SimpleLineSymbol lineSymbol;
+  private SimpleLineSymbol polygonLineSymbol;
+  private SimpleMarkerSymbol pointSymbol;
+  private Graphic graphic;
+  private ComboBox<SketchCreationMode> sketchComboBox;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+      scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Sketch on Map Sample");
+      stage.setWidth(1000);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map with a basemap and add it to the map view
+      ArcGISMap map = new ArcGISMap(Basemap.Type.IMAGERY, 64.3286, -15.5314, 13);
+      mapView = new MapView();
+      mapView.setMap(map);
+
+      // create a graphics overlay for the graphics
+      graphicsOverlay = new GraphicsOverlay();
+
+      // add the graphics overlay to the map view
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a new sketch editor and add it to the map view
+      sketchEditor = new SketchEditor();
+      mapView.setSketchEditor(sketchEditor);
+
+      // define symbols for graphics
+      pointSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.SQUARE, 0xFFFF0000, 20);
+      lineSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, 0xFF64c113, 4);
```

Comment:
A comment above each symbol describing the color would be helpful: "thin green line"